MovieGallery/Pages/Search.cshtml.cs

```diff
@@ -22,52 +22,60 @@ public class SearchModel : PageModel
 
         public void OnGet(string SearchbyYear)
         {
-            using (StreamReader r = new StreamReader("Movies.json"))
+            try
```

Comment:
Added try-catch statement for exception handling according to best practices source:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions